### ISSUE TITLE
API Compat: the reference assembly is allowed to have more overrides than the implementation

### ISSUE THREAD

**Issue by terrajobst** [dotnet/arcade#2570]
Suppos a reference assembly includes an overide that the implementation doesn't have. Then, API Compat creates an error like this:

```
MembersMustExist : Member 'System.IO.FileStream.Read(System.Span<System.Byte>)' does not exist in the implementation but it does exist in the contract.
```

However, IL was designed to handle cases like this. In fact, we may want to always have all the overrides to bad `base` calls in C# should we add an overide later.

See [this thread](https://github.com/dotnet/standard/pull/1158#discussion_r276904337) for more context.

**Comment by markwilkie** [comment]
cc/ @ericstj

**Comment by ericstj** [comment]
Yeah, this is an ask to relax a rule in APICompat.  Any chance we can have labels to triage this sort of thing @markwilkie?

**Comment by terrajobst** [comment]
Also, should we archive dotnet/build tools now?

**Comment by markwilkie** [comment]
If a label is needed, I think that's fine.  We use epic assigned via ZenHub which I'd prefer, but I understand if that's not really tenable.

**Comment by ericstj** [comment]
We should look more closely at the override behavior in general.  Contact can have more overrides than implementation.  If the implementation has an override not in the contract we may want to flag that as well for the same reason @terrajobst mentioned.

**Comment by ViktorHofer** [comment]
Closing as APICompat got removed from arcade, rewritten and now lives in dotnet/sdk: https://github.com/dotnet/sdk/tree/main/src/Compatibility
